src/bundle/Resources/views/themes/admin/content/form_fields.html.twig

```diff
@@ -48,17 +48,17 @@
 
 {% block form_row_subfield -%}
     {% set wrapper_class = 'ibexa-data-source__field ibexa-data-source__field--' ~ name %}
-    {% if required %}{% set wrapper_class = (wrapper_class ~ ' ibexa-data-source__field--required ibexa-data-source__field--required')|trim %}{% endif %}
+    {% if required %}{% set wrapper_class = (wrapper_class ~ ' ibexa-data-source__field--required')|trim %}{% endif %}
     {% if errors|length > 0 %}{% set wrapper_class = (wrapper_class|default('') ~ ' is-invalid')|trim %}{% endif %}
 
-    {% set label_attr = label_attr|merge({'class': (label_attr.class|default('') ~ ' ibexa-label ibexa-data-source__label ibexa-data-source__label')|trim}) %}
-    {% set label_wrapper_attr = label_wrapper_attr|default({})|merge({'class': (label_wrapper_attr.class|default('') ~ ' ibexa-data-source__label-wrapper ibexa-data-source__label-wrapper')|trim}) %}
+    {% set label_attr = label_attr|merge({'class': (label_attr.class|default('') ~ ' ibexa-label ibexa-data-source__label')|trim}) %}
```

Comment:
I'm not sure whether we should delete these duplicates. When we try to trace back where they come from we can see that first it was duplicated to have both `ibexa-` and `ez-`.
1. https://github.com/ibexa/admin-ui/commit/c5cb76b4a7e854965695c55a994305caf759f230#diff-3776f7fe593a6a85de0c62980e813efe1256ea7398646bff8c34a16ad83d91aaR45
2. and then `ez-` was probably changed to `ibexa-` by a mistake in this PR: 
https://github.com/ibexa/admin-ui/commit/a4bdc57fc057c6e85f823292fb8299b5ef90d00a#diff-3776f7fe593a6a85de0c62980e813efe1256ea7398646bff8c34a16ad83d91aaR47 

So probably we should revert duplicates to `ez-` to revert changes from 2nd PR? But maybe we no longer want to have `ez-`, especially given we have already version `4.5`?
```suggestion
    {% if required %}{% set wrapper_class = (wrapper_class ~ ' ibexa-data-source__field--required ez-data-source__field--required')|trim %}{% endif %}
```

Reply:
As main branch currently points to 5.0 we can safely forget about `ez-` counterpart. @tischsoic Could you please take over this PR ?